build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     }
     dependencies {
         // wasn't working with travis
```

Comment:
hmm.  what's up with this comment?

Reply:
Please remove. This was when the beta build tool wasn't available in Travis and I was falling back to an old version